stories/Atom/Global/Icons/icons.scss

```diff
@@ -0,0 +1,41 @@
+@import "../../../assets/scss/variables";
+
+/* icon start */ 
+.icon--bar {
+  cursor: pointer;
+}
+.arrow-expand {
+  height: 16px;
+  cursor: pointer;
+  background: url(../../../assets/images/Icon/arrow-left-expand-defaut.svg) no-repeat center center  ;
```

Comment:
@harishkumar-srijan can we create the variable for the images path so that we can use wherever we use the image in css

Reply:
Done